### PR TITLE
[Keyboard] Add Piantor UV 44

### PR DESCRIPTION
**Pull request by azhizhinov**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/piantoruv44/readme.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Piantor UV 44
+
+![Piantor UV 44](https://github.com/user-attachments/assets/43d3886e-2063-45a0-89b7-176b8744ccfb)
```
Comment (waffle87) [inline keyboards/piantoruv44/readme.md:3]: Could this image be rehosted on Imgur.

**Comment by azhizhinov** [comment]
Hello. Changed to ImgBB, is that OK?

**Review by waffle87 — APPROVED** [review]

**Comment by drashna** [comment]
As of August 26, 2024, `qmk/qmk_firmware` is no longer accepting VIA-enabled keymaps as these have now transitioned to a repository under the VIA team's control.

As you've submitted a PR containing `via` or VIA-enabled keymap(s), this is your notice that they should be removed from this PR. You should now submit a secondary PR to the [VIA QMK Userspace](https://github.com/the-via/qmk_userspace_via) repository with your associated `via` or VIA-enabled keymaps instead.

Otherwise, LGTM.

File: keyboards/piantoruv44/rules.mk
Comment (drashna) [inline keyboards/piantoruv44/rules.mk:1]: This can be removed.

File: keyboards/piantoruv44/keyboard.json [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        "matrix_pins": {
+            "right": {
+                "direct": [
+                    ["GP2", "GP3", "GP15", "GP11", "GP4", "GP5"],
+                    ["GP8", "GP9", "GP14", "GP10", "GP20", "GP22"],
+                    ["GP12", "GP13", "GP7", "GP6", "GP19", "GP21"],
+                    ["GP26", "GP16", "GP18", "GP17", null, null]
+                ]
+            }
+        }
```
Comment (drashna) [inline keyboards/piantoruv44/keyboard.json:43]:
```suggestion
        },
        "serial": {
            "driver": "vendor"
        }
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by azhizhinov** [comment]
@drashna 
Hello.
Done.

**Comment by zvecr** [comment]
> Hello.
Done.

The following have **not** been completed.
https://github.com/qmk/qmk_firmware/pull/24247#discussion_r1737750106
https://github.com/qmk/qmk_firmware/pull/24247#discussion_r1737731357

**Comment by azhizhinov** [comment]
@zvecr 
Hello.
Done.

**Comment by zvecr** [comment]
https://github.com/qmk/qmk_firmware/pull/24247#discussion_r1737731357

...nope

**Comment by azhizhinov** [comment]
@zvecr 
rules.mk has been deleted.

**Comment by azhizhinov** [comment]
Hello.
@drashna, sorry to bother you, but could you review this PR?
Is anything there for me to improve?
Thanks in advance.

File: keyboards/piantoruv44/readme.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Piantor UV 44
+
+![Piantor UV 44](https://i.ibb.co/QvJM1GR/20240806-PIANTORUV44-00.png)
```
Comment (drashna) [inline keyboards/piantoruv44/readme.md:3]: This image doesn't load?

Comment (fauxpark) [reply keyboards/piantoruv44/readme.md:3]: Loads for me. Imgur is preferred though.

File: keyboards/piantoruv44/keyboard.json
Comment (fauxpark) [inline keyboards/piantoruv44/keyboard.json:1]: Please run `qmk format-json -i` on this file.

File: keyboards/piantoruv44/readme.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+* Hardware Supported: RP2040
+* Hardware Availability: https://github.com/azhizhinov/PIANTORUV44
+
+Make example for this keyboard (after setting up your build environment):
+
+    make azhizhinov/piantoruv44:default
+
+Flashing example for this keyboard:
+
+    make azhizhinov/piantoruv44:default:flash
```
Comment (fauxpark) [inline keyboards/piantoruv44/readme.md:17]:
```suggestion
    make piantoruv44:default

Flashing example for this keyboard:

    make piantoruv44:default:flash
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/piantoruv44/readme.md (hunk 3 above)
Comment (azhizhinov) [reply keyboards/piantoruv44/readme.md:3]: Hello.
It's perfectly fine.
Снимок экрана от 2024-09-29 10-20-47

File: keyboards/piantoruv44/keyboard.json
Comment (azhizhinov) [reply keyboards/piantoruv44/keyboard.json:1]: Hello.
@fauxpark 
Done.
Thanks in advance.

**Review by drashna — APPROVED** [review]

**Pull request merged**